src/Utils.php

```diff
@@ -67,4 +68,15 @@ public static function confirmation( $confirmation ) {
 
 		return $confirmation;
 	}
+
+	/**
+	 * Delete the form entry after the form is submitted.
+	 * Usefull for Login/Signup forms, because we don't want
+	 * to store plain text passwords in the database as an entry.
+	 *
+	 * @param object $entry Gravity Forms entry.
+	 */
+	public static function delete_entries( $entry ) {
+		GFAPI::delete_entry( $entry['id'] );
```

Comment:
Why is that? this comes from a Gravity Form filter, how can be possible that we don't have the id inside? @mitogh